scripts/database/mysqlConnect.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+# coding=utf-8
+
+import sys, os, inspect, configparser
+from util import configSectionMap
+from peewee import *
+
+# read database config file
+config = configparser.ConfigParser()
+dbDir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+config.read(dbDir + '/config.ini')
+
+ConfigMap = configSectionMap('DB', config)
+mHost = str(ConfigMap['host'])
+mPort = int(ConfigMap['port'])
+mUser = str(ConfigMap['user'])
+mPasswd = str(ConfigMap['passwd'])
+mDb = str(ConfigMap['db'])
+
+conn = MySQLDatabase(mDb, host = mHost, port = mPort, user = mUser, passwd = mPasswd)
+
+class BaseModel(Model):
+    class Meta:
+        database = conn
+
+class pornVideo(BaseModel):
+    id = AutoField()
+    source = CharField()
+    view_numbers = IntegerField()
+    video_id = CharField()
+    view_ratings = CharField()
+    video_title = CharField()
+    create_date = DateTimeField()
+
+    class Meta:
+        table_name = 'porn_videos'
+
```

Comment:
I think it will be good to split these classes from this Python file.

I prefer one class has one Python file :).

Reply:
And keep these classes following single responsibility :).

Reply:
This is follow the example from peewee document.
http://docs.peewee-orm.com/en/latest/peewee/models.html

I will try to split them to independent file, but I can't promise i can do it.
So, maybe we can open another pull request to implement it.

Reply:
Ok. Don't forget to do this on upcoming PR!